_includes/head.html

```diff
@@ -30,24 +25,29 @@
 
 <link rel="shortcut icon" href="{{ "images/favicon.ico"  }}">
 
+
 <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
```

Comment:
We don't support IE 8 or 9. Mind ripping out the comments and the two IE8 scripts.